.circleci/config.yml

```diff
@@ -91,6 +91,9 @@ jobs:
       JUNIT_REPORT_PATH: coverage/junit/unit_tests
     steps:
       - *attach_workspace
+      - run:
+          name: Store Unit Tests
+          command: yarn nx test store --configuration=ci --verbose
```

Comment:
Let's remove this again
```suggestion
```

Reply:
this one will still fail, because I haven't changed configurations, only an actual test command for circle ci. I'm not really positive to define amount of workers within jest.config as Jest is usually smart enough to identify amount of CPUs properly (at least it works locally and on GHA)